packages/nextjs/services/web3/connectors.tsx

```diff
@@ -18,7 +29,7 @@ function getConnectors() {
     connectors.push(new BurnerConnector());
   }
 
-  return connectors.sort(() => Math.random() - 0.5);
+  return connectors.sort(() => Math.random() - 0.5).map(withDisconnectWrapper);
```

Comment:
It will be also a great chance to refactor this sort using Math.random() - 0.5, what do you think? @jrcarlos2000

Reply:
are we considering to use `lodash` ?

Reply:
@metalboyrick i think something like [this](https://github.com/Quantum3-Labs/scaffold-stark-2/pull/152/files) ?

Reply:
this connectors file was updated when we tried to have a conditional connector list, that doesnt show the burner connector when the target network is not `devnet`.

Reply:
if its possible to go back to use [this](https://github.com/apibara/starknet-react/blob/852f202a48e6a347ffa950fd4fb2c794702699b6/packages/core/src/connectors/discovery.ts#L21) hook from `starknet-react` , wdyt ? @metalboyrick

Reply:
better make new pr i think, refactoring change

Reply:
options: either to use lodash shuffle or implement our own array shuffles, or use this hook, i think both works. Either way we still need the disconnect wrapper to ensure that wallet disconnects AND removes the local storage entry

Reply:
just checked the `useInjectedConnector` from starknet react, they use `lodash` shuffle under the hood haha.